Resources/Private/JavaScript/Host/Redux/CR/ContentDimensions/index.js

```diff
@@ -0,0 +1,81 @@
+import {createAction} from 'redux-actions';
+import Immutable, {List, Map} from 'immutable';
+import {$set, $get} from 'plow-js';
+
+import {handleActions} from 'Shared/Utilities/index';
+import {actionTypes as system} from 'Host/Redux/System/index';
+import {createSelector} from 'reselect';
+
+const SELECT_PRESET = '@neos/neos-ui/CR/ContentDimensions/SELECT_PRESET';
+const SET_ACTIVE = '@neos/neos-ui/CR/ContentDimensions/SET_ACTIVE';
+
+export const actionTypes = {
+    SELECT_PRESET,
+    SET_ACTIVE
+};
+
+/**
+ * Selects a preset for the given content dimension
+ */
+const selectPreset = createAction(SELECT_PRESET, (name, presetName) => ({name, presetName}));
+
+/**
+ * Sets the currently active content dimensions
+ */
+const setActive = createAction(SET_ACTIVE, dimensionValues => ({dimensionValues}));
+
+//
+// Export the actions
+//
+export const actions = {
+    selectPreset,
+    setActive
+};
+
+const active = $get('cr.contentDimensions.active');
+const byName = $get('cr.contentDimensions.byName');
+const allowedPresets = $get('cr.contentDimensions.allowedPresets');
+
+//
+// Export the reducer
+//
+export const reducer = handleActions({
+    [system.INIT]: state => $set(
+        'cr.contentDimensions',
+        new Map({
+            byName: Immutable.fromJS(byName(state)),
+            active: Immutable.fromJS(active(state)),
+            allowedPresets: Immutable.fromJS(allowedPresets(state))
```

Comment:
Here, I'd like to have some documentation how the structure in the state actually looks like :-) As that is crucial to understand for the other parts